less/viewdetails.less

```diff
@@ -178,6 +178,31 @@
     }
   }
 
+  .md-alert {
+    padding: 15px;
+    margin-bottom: 20px;
+    border: 1px solid transparent;
+    border-radius: 1px;
+  }
+
+  .md-alert-info {
+    border-color: #bce8f1;
```

Comment:
Are you re-using colors already defined in the CSS, or simply the bootstrap one? In all cases, I suggest we store them in variables.

Reply:
Simply use the bootstrap one

Reply:
Then, here again, let it this way. But in the medium term, we should have better CSS organisation.